bnb-chain/bnc-tendermint
# [R4R] #27 eventbus and indexservice start before handshaker

#27 
tendermint one: https://github.com/tendermint/tendermint/pull/3194

This is manually tested in BinanceChain:
1. start order gen
2. set breakpoint on EndBlocker
3. close chain process when breakpoint is hit simulate endblocker panic
4. restart chain process
5. ensure query tx for panic height can see the tx

* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Updated CHANGELOG_PENDING.md
